tile_fetch.py

```diff
@@ -51,7 +61,8 @@ def __init__(self, url):
         self.token = token or b''
         url_path = urllib.parse.unquote_plus(urllib.parse.urlparse(url).path)
         self.image_slug, image_id = url_path.split('/')[-2:]
-        self.image_name = '%s - %s' % (string.capwords(self.image_slug.replace("-"," ")), image_id)
+        self.image_name = unidecode.unidecode(string.capwords(self.image_slug.replace("-"," ")))
```

Comment:
Why do we need to remove non-ascii characters here ?

Reply:
I believe I ran into some errors where the filename contained non-ascii characters, and thus download wouldn't finalize on Windows. By replacing non-ascii characters it becomes system compliant.

Reply:
What characters exactly ? All major operating systems accept unicode characters in filenames. And on the other hand, there are many ASCII characters that are rejected on windows. Can we just strip invalid characters from the name ?

Reply:
It was a Spanish accent name, one of [these "e"s](https://www.spanishdict.com/answers/282262/how-to-type-spanish-letters-and-accents-from-paralee). I don't have the URL on hand. I would avoid stripping those letters from the name - it's part of the actual name. Replacing it with a like-letter such as "e" seems to be a better solution.

Reply:
Accented letters can be parts of filenames in all operating systems

Reply:
That is weird, because I'm believe [this link](https://artsandculture.google.com/asset/cabeza-de-hierro-jaume-morera-gal%C3%ADcia/ZQGdSYXWL7WZUw) was running into issues without this snippet of code. I believe it was the letter "í".